<launch>
	<!--include file="$(find master_project)/launch/turtlebot3_search_and_rescue.launch"/-->

	<include file="$(find master_project)/launch/start_rossharp_connection.launch"/>
	<include file="$(find master_project)/launch/global_planner_custom.launch"/>
	
	<!--node pkg="web_video_server" type="web_video_server" name="web_video_server" output="screen"/-->
	<!--node pkg="rosbridge_server" type="rosbridge_websocket.py" name="rosbridge_websocket" output="screen"/-->
	
	<node pkg="master_project" type="generate_image_from_odom_occupancy.py" name="map_creator" output="screen"/>
	<node pkg="master_project" type="resize_image_occupancy.py" name="visual_adaptation" output="screen"/>
	<node pkg="master_project" type="publish_list_test.py" name="goal_sender" output="screen"/>
	<node pkg="master_project" type="global_planner_script.py" name="path_planner" output="screen"/>
	<!--
	<arg name="gui" default="true"/>
	<arg name="image_topic" default="/camera/rgb/image_raw"/>
	<arg name="in" default="/visual_augmented_image"/>
	<arg name="out" default="/visual_augmented_image"/>

	<node name="find_object_2d" pkg="find_object_2d" type="find_object_2d" output="screen">
		<remap from="image" to="$(arg image_topic)"/>
		<param name="gui" value="$(arg gui)" type="bool"/>
		<param name="in" value="$(arg in)" type="str"/>
		<param name="out" value="$(arg out)" type="str"/>
	</node>
	-->
</launch>